### ISSUE TITLE
core.css Doesn't Style Top Left Drop-Down Menu

### ISSUE THREAD

**Issue by zyigo** [calrobertlee/roam-css-styles#33]
## Overview
Found some issues with styling of the Drop down menu. It's not styled what so ever, making the text unreadable in dark themes. 

## Proposed Solution
My temporary solution is the following (tested). I didn't raise a pull request in case you wanted to do it your way.
``` CSS
.bp3-menu-item.menu-item {
    color: var(--sidebar-font-color);
}

.bp3-menu-item.menu-item:hover {
    color: var(--popover-font-color);
}
```

## Additional Note
The "Graphs & Settings" button doesn't play nice with this, and wasn't fixed with my temporary solutions. Perhaps you can get it all in one go?

Cheers,
Morgan

**Comment by zyigo** [comment]
Noticed you updated this in recent commit! Thanks Cal!
